content/webinar/_index.md

```diff
@@ -1,5 +1,5 @@
 ---
-private: true
 title: Webinars
```

Comment:
I guess these are existing at https://www.pulumi.com/webinar/, but is there any reason not to make the URL plural? `/webinars` instead of `/webinar`

Similar to how we have `/events`.

We could forward the old pages to the new ones.

Reply:
I made the change to use `/webinars` instead of `/webinar` right now things are forwarded via Hugo but we should probably get these into REDIRECTRON 9000. I filed an issue here: https://github.com/pulumi/home/issues/719

Reply:
We shouldn't need to use the redirectron for this. We automatically convert HTML meta-redirects to proper 301 redirects when deploying the site.

https://github.com/pulumi/docs/blob/6c5bb98f1b3e506bdcc3cc17f1879b8cf6e42c7c/infrastructure/index.ts#L456-L466

https://github.com/pulumi/docs/blob/6c5bb98f1b3e506bdcc3cc17f1879b8cf6e42c7c/infrastructure/sync.sh#L56-L63